packages/@expo/cli/src/export/exportApp.ts

```diff
@@ -137,6 +134,9 @@ export async function exportAppAsync(
 
   if (platforms.includes('web')) {
     if (useServerRendering) {
+      // TODO: Remove when this is abstracted into the files map
+      await copyPublicFolderAsync(publicPath, path.resolve(outputPath, 'client'));
```

Comment:
This branch is called for both output static and server, but the copy is only needed for output server